src/ngx_http_lua_headers_in.c

```diff
@@ -645,14 +645,95 @@ ngx_http_clear_builtin_header(ngx_http_request_t *r,
 }
 
 
+static ngx_hash_init_t  lua_req_header_hash;
+static u_char           header_string[512];
```

Comment:
if you agree, I can pass down the lmcf as argument by this path: ngx_http_lua_inject_req_api -> ngx_http_lua_inject_req_header_api -> lua_req_set_header_hash_init